polling_stations/apps/data_importers/import_script.py

```diff
@@ -47,4 +60,4 @@ class Command({self.importer_class}):
     stations_name = "{self.stations_name}"
     elections = {self.elections}{self.csv_encoding_string}{self.csv_delimiter_string}
 """
-        return black.format_str(script_str, mode=black.FileMode())
+        return ruff_format_str(script_str)
```

Comment:
Well caught. This has been the bit of friction that stopped me getting rid of black.